### PR TITLE
coq-flocq-quickchick update for coq 8.11.0

### PR DESCRIPTION
**Pull request by p3rsik**

### CODE REVIEW

File: released/packages/coq-flocq-quickchick/coq-flocq-quickchick.1.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+name: "coq-flocq-quickchick"
+version: "1.0.2"
```
Comment (clarus) [inline released/packages/coq-flocq-quickchick/coq-flocq-quickchick.1.0.2/opam:3]: Name and version are redundant.

Comment (clarus) [reply released/packages/coq-flocq-quickchick/coq-flocq-quickchick.1.0.2/opam:3]: Redundant with the file name.

Comment (p3rsik) [reply released/packages/coq-flocq-quickchick/coq-flocq-quickchick.1.0.2/opam:3]: **with the file name** you mean, that if I rename `opam` file to `coq-flocq-quickchick.1.0.2.opam` I can omit this fields?

**Comment by p3rsik** [comment]
`[ERROR] Package coq-flocq-quickchick has no version 1.0.2.` help me, please, I don't understand what this means(this is from checks)

File: released/packages/coq-flocq-quickchick/coq-flocq-quickchick.1.0.2/opam (hunk 1 above)
Comment (p3rsik) [reply released/packages/coq-flocq-quickchick/coq-flocq-quickchick.1.0.2/opam:3]: Okay, I looked at another package opam file and figured it out myself, thank you

File: released/packages/coq-flocq-quickchick/coq-flocq-quickchick.1.0.2/opam [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "coq" {= "8.11.0"}
+  "coq-quickchick" {>= "1.3.0"}
+  "coq-flocq" {>= "3.2.0"}
+]
+build: [
+  ["coq_makefile" "-f" "_CoqProject"  "-o" "Makefile"]
+  [make]
+]
+install: [make "install"]
+dev-repo: "git+https://github.com/digamma-ai/flocq-quickchick.git"
```
Comment (clarus) [inline released/packages/coq-flocq-quickchick/coq-flocq-quickchick.1.0.2/opam:18]:
```suggestion
dev-repo: "git+https://github.com/digamma-ai/flocq-quickchick.git"
tags: [
  "logpath: FlocqQuickChick"
  "date: 2020-04-01"
]
```

**Comment by clarus** [comment]
> `[ERROR] Package coq-flocq-quickchick has no version 1.0.2.` help me, please, I don't understand what this means(this is from checks)

It seems that it works now.

**Pull request merged**